cesium/internal/domain/writer.go

```diff
@@ -303,10 +311,10 @@ func (w *Writer) Close(ctx context.Context) error {
 
 func (w *Writer) validateCommitRange(end telem.TimeStamp, switchingFile bool) error {
 	if !w.prevCommit.IsZero() && !switchingFile && end.Before(w.prevCommit) {
-		return errors.Wrap(validate.Error, "commit timestamp must not be less than the previous commit")
+		return errors.Wrapf(validate.Error, "commit timestamp %s must not be less than the previous commit timestamp %s", end, w.prevCommit)
 	}
 	if !w.Start.Before(end) {
-		return errors.Wrap(validate.Error, "commit timestamp must be strictly greater than the starting timestamp")
+		return errors.Wrapf(validate.Error, "commit timestamp %s must be strictly greater than the starting timestamp %s", end, w.Start)
```

Comment:
We get this error all the time when dealing with timing issues on the driver side. It'd be good to represent these as actual stringified timestamps. And then it would also be good to indicate what the delta between the timestamps are (as a nicely strinigified timespan). See how we do this in [typescript](https://github.com/synnaxlabs/synnax/blob/3aac1847a5d1411c4204e0a3166f3bee63aad733/x/ts/src/telem/telem.ts#L507). It has been very useful